src/libs/ReactNativeLauncher.js

```diff
@@ -57,7 +58,8 @@ class ReactNativeLauncher extends Launcher {
     this.controller = new AbortController()
 
     const wrapTimer = wrapTimerFactory({
-      logFn: msg => this.log({ level: 'info', msg })
+      logFn: msg =>
+        this.log({ level: 'info', msg, timestamp: new Date().toISOString() })
```

Comment:
nit pour l'avenir: est-ce qu'on ne devrait pas mettre ce timestamp en fallback value dans `this.log()` pour être sûr de tout le temps en avoir un même si le développeur oublie d'en mettre un?

Reply:
Done